app/controllers/users.js

```diff
@@ -269,3 +271,68 @@ exports.user = (req, res, next, id) => {
       next();
     });
 };
+
+/**
+ * Invite user to play game.
+ */
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey('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');
+
+exports.invite = (req, res) => {
+  const { recieverEmail, gameURL } = req.body;
+  const { name } = req;
+  const msg = {
+    from: 'cfh@andela.com', // sender address
+    to: recieverEmail, // list of receivers
+    subject: `${name} is inviting you to join a game`, // Subject line
+    html: `<h1>Testing</h1><p>Join the game ${gameURL}</p>` // plain text body
+  };
```

Comment:
The comments on each line of this object is a bit unnecessary let's get rid of it. @Rhotimee @OlatunjiYso